src/app/modals/property-management-modal/property-management-modal.component.ts

```diff
@@ -1,6 +1,8 @@
 import { Component, OnInit } from '@angular/core';
 import { propertyForm } from '../../../form/form.service';
 import { NgbModal, NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
```

Comment:
NgbModal is being used?

Reply:
No, as well as other imports in the component. Should i remove them?

Reply:
Yes please.

Reply:
Done